dockstore-webservice/src/main/resources/openapi3/openapi.yaml

```diff
@@ -4550,6 +4547,29 @@ paths:
       tags:
         - organizations
   /organizations/{organizationId}:
+    delete:
+      description: Delete pending or rejected organization
+      operationId: deleteRejectedOrPendingOrganization
+      parameters:
+        - description: Organization ID.
+          in: path
+          name: organizationId
+          required: true
+          schema:
+            format: int64
+            type: integer
+      responses:
+        "204":
+          description: OK
```

Comment:
200 is OK,
204 is No Content

Reply:
Fixed. Thought 204 also outputted OK but with no content.